internal/contracts/manager.go

```diff
@@ -53,6 +53,7 @@ type Manager interface {
 	GetContractSubscriptionByNameOrID(ctx context.Context, ns, nameOrID string) (*fftypes.ContractSubscription, error)
 	GetContractSubscriptions(ctx context.Context, ns string, filter database.AndFilter) ([]*fftypes.ContractSubscription, *database.FilterResult, error)
 	DeleteContractSubscriptionByNameOrID(ctx context.Context, ns, nameOrID string) error
+	GenerateFFI(ctx context.Context, ns, name, version string, input *fftypes.JSONAny) (*fftypes.FFI, error)
```

Comment:
Any reason we pass name and version here? Seems like the only usage of this method always sets them to empty strings.

Reply:
Yeah... maybe worth a quick chat about this one... I wrote the backend first and thought that it would make sense to pass in the name, version, etc. so it could be filled in on the generated FFI. Then I created the API endpoint and couldn't think of a good place to specify them when calling the API, because the payload of that request right now is entirely an ABI array.

One option would be to modify the endpoint to wrap the ABI elements in another object so that request would look like:

```json
{
  "name": "...",
  "version": "...",
  "abi": [...]
}
```